benchmarks/instruction_counts/definitions/standard.py

```diff
@@ -51,17 +51,79 @@
     },
 
     "Pointwise": {
-        "Math": {
-            "add": {
-                "Tensor-Scalar": GroupedStmts(
-                    r"x += 1.0",
-                    r"x += 1.0;",
-                    setup=Setup.GENERIC.value,
-                ),
-            },
-        },
+        "Math": GroupedVariants(*parse_stmts(r"""
+            Python                                   | C++
+            ---------------------------------------- | ----------------------------------------
```

Comment:
It seems like it'll be pretty difficult to tell if you have sufficient coverage of framework code with the benchmarks. Which kind of suggests some sort of way of running coverage on the performance tests might be a good idea.

Reply:
Can we defer this to a follow up? I think we need to put some thought into exactly what a useful definition of "coverage" would be.

Reply:
Yes, please defer